README.md

```diff
@@ -56,13 +56,17 @@ fiddle has grown up, export it as a project with or without
 [electron-forge][electron-forge]. Then, use your favorite editor and take on
 the world!
 
-### Running Fiddle
+### Building Fiddle
 
 1. Clone this repository: `git clone https://github.com/electron/fiddle.git`.
 2. Change current directory to fiddle: `cd ./fiddle`
 3. Run `npm install`
 4. Run `npm start`
 
+### Running Fiddle
+
+Download and run the latest binary from the electron [releases page](https://electronjs.org/releases/stable).
+
```

Comment:
I don't think this is necessary, since there's already a Download link after the first paragraph in the README.

Reply:
okay, removing this and placing the build instructions in contributing.md. 

One quick question, should I put a generic contribution guide or would one/more of the members do it?

Reply:
I think a contributor's guide is a wonderful idea!